content/swiftui/concepts/animation/animation.md

```diff
@@ -0,0 +1,107 @@
+---
+Title: 'Animation'
+Description: 'In response to state changes, produce seamless visual updates.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'iOS'
+CatalogContent:
+  - 'learn-iOS-course'
+  - 'path/iOS-path'
+---
+
+**Animation** is the ability to add smooth view changes to your view by changing the state of the view. There is two types of animation in swiftUI: Implicit animation and Explicit animation both of them will be explained in the sections below. it's very important to add animation to the user experience of an app, because it makes the app more interactive with the user and enhance the user experience of the app. And also Animating custom views can be done by conforming the view to the `Animatable` protocol, and telling SwiftUI about the value you want to animate.
```

Comment:
[Proofreading] While mentioning the two types of animation is helpful, it would be best to just introduce them in the other sections and mostly focus on defining animations in this section.   
```suggestion
**Animation** is the ability to add smooth view changes to your view by changing the state of the view.  Adding animations to the app makes it more interactive, enhancing user experience.
```